File: zig/001.zig
// Q: Add all the natural numbers below one thousand that are multiples of 3 or 5.
// A: 233168

const print = @import("std").debug.print;

pub fn main() void {
    var i: u16 = 0;
    var sum: u32 = 0;

    while (i < 1000) : (i += 1) {
        if (i % 3 == 0 or i % 5 == 0)
            sum += i;
    }

    print("{d}\n", .{sum});
}

File: golang/001.go
// Q: Add all the natural numbers below one thousand that are multiples of 3 or 5.
// A: 233168

package main

import "fmt"

func main() {
	sum := 0

	for i := 0; i < 1000; i++ {
		if i%3 == 0 || i%5 == 0 {
			sum += i
		}
	}

	fmt.Println(sum)
}
